tf-repo-mgmt/scripts/New-Repository.ps1

```diff
@@ -61,7 +67,7 @@ if (!$skipMetaDataCreation) {
   git commit -m "chore: add $moduleName metadata"
   git push --set-upstream origin "chore/add/$moduleName"
 
-  $prUrl = gh pr create --title "chore: add $moduleName metadata" --body "This PR adds metadata for the $moduleName module." --base main --head "chore/add/$moduleName" -a "@me" --repo $governanceRepoUrl
+  $prUrl = gh pr create --title "chore: add $moduleName metadata" --body "This PR adds metadata for the $moduleName module."
```

Comment:
Doesn't the PR command need to know the branches / fork? Or does it default somehow?

Reply:
It uses the current branch for the head, and the default branch of the default repo, as configured by the `gh repo set-default`.

I'll run through this using the switches to disable the repo & issue creation